Assets/Scripts/UI/EMCli.cs

```diff
@@ -0,0 +1,184 @@
+﻿using UnityEngine;
+using System.Collections;
+using System;
+using Assets.Scripts.UI;
+using UnityEngine.SceneManagement;
+
+public class EMCli : MonoBehaviour
+{
+    private static Rect windowRect = new Rect(Screen.width - 320f, 0, 300f, 430f);
+    private static GUILayoutOption[] options1 = new GUILayoutOption[] { GUILayout.Width(320f), GUILayout.MaxHeight(430f) };
+    private static GUI.WindowFunction func1 = new GUI.WindowFunction(WindowLayout);
```

Comment:
Same as above, but name it windowFunction as func could be confused with Func<T1, TOut>